SpringMVCAnnotationOnlineStore/src/main/java/org/o7planning/springmvconlinestore/model/CartInfo.java

```diff
@@ -107,6 +109,15 @@ public double getAmountTotal() {
         }
         return total;
     }
+    
+    public double getFinalizedTotal(Double shipping) {
```

Comment:
What happens if a customer clicks 'continue shopping' from the cart and adds a second product instead of incrementing the 'quantity'?

Reply:
A valid question. I just checked again to make sure my logic held, and the getFinalizedTotal() method call updates every time a user hits the shoppingCartConfirmation page. So I walked through the case you mentioned above, and if the user decides to continue shopping, they can add or take away as much as they want. And when they hit the shoppingCartConfirmation page (step right before the order is created) they will see an updated total representing their latest cart state. 
Is this how you anticipated it to work?